master/buildbot/www/oauth2.py

```diff
@@ -81,13 +81,13 @@ class OAuth2Auth(auth.AuthBase):
     tokenUriAdditionalParams = {}
     loginUri = None
     homeUri = None
-    sslVerify = None
 
-    def __init__(self, clientId, clientSecret, autologin=False, **kwargs):
+    def __init__(self, clientId, clientSecret, autologin=False, sslVerify=True, **kwargs):
```

Comment:
Not mandatory for the PR but nice to have:

```suggestion
    def __init__(self, clientId: str, clientSecret: str, autologin: bool = False, sslVerify: bool = True, **kwargs: Any) -> None:
```

Reply:
Would also need a import of Any at the top.

Reply:
Thanks for suggestions, I improved the commit.
I only introduced the type for the argument which I added (ssl_verify).  Specifying other types could potentially break the code as I did not specifically analyze the types of those arguments. Maybe in next PR.